doc/reference/cardano-governance.md

```diff
@@ -0,0 +1,235 @@
+# Cardano governance
+
+Until the [Voltaire era](https://roadmap.cardano.org/en/voltaire/) is released, Cardano has a federated governance mechanism that allows updating the protocol parameters. Such updates are done using an _update proposal_ mechanism. In the current setup, only the holders of the _genesis keys_ can submit and vote on proposals.
+
+Update proposals can be used to introduce changes to the values of specific parameters (e.g., the maximum block size, the maximum transaction size etc.), changes to the validation rules at any level (transaction, block, or blockchain), or even changes at the consensus protocol itself.
+
+Protocol parameters are used to affect the operation of the Cardano protocol. They may be either **updatable** or **non-updatable**. Updatable parameters can be tuned to vary the operation of the block-producing protocol, impacting the proportion of pools that are federated/non-federated, influence of the 'pledge', etc. Non-updatable parameters affect the fundamentals of the blockchain protocol, which includes defining the genesis block, basic security properties etc. [CIP-9](https://cips.cardano.org/cips/cip9/) and [CIP-28](https://cips.cardano.org/cips/cip28/) detail which parameters are updatable and which are not.
+
+### Cardano eras
+
+Cardano has come through different development **eras**, each of which introduced new functionalities. Going from one era to the next requires that new features (which might require changes in the ledger, consensus, network, etc) are incorporated into a new node version. For this, an _update proposal_ must be submitted to trigger the update which is handled by the _hardfork combinator_.
```

Comment:
do you want to mention/link [CIP-59](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0059)? it tries to be clear about what "era" means.